### DISCUSSION TITLE
No RF Comands on Sonoff 4CH Pro RF Rev3

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#15354] [Q&A] [answered]
I flashed tasmota 11.0.0(sensors) on Sonoff 4CH Pro RF Rev3.
{"NAME":"Sonoff 4CH Pro","GPIO":[32,1,1,1,226,225,33,34,224,320,35,227,0,0],"FLAG":0,"BASE":23}

I tried to explore the rf functionallity.
No rf commands are known:

01:18:19.375 CMD: RfProtocol 
01:18:19.383 MQT: stat/tasmota_736697/RESULT = {"Command":"Unknown"}
01:18:21.883 CMD: Rfsend
01:18:21.890 MQT: stat/tasmota_736697/RESULT = {"Command":"Unknown"}

I tried with and without wired connection from STM to GPIO2 as shown [here](https://tasmota.github.io/docs/devices/Sonoff-4CH-Pro/#sending-rf-codes). Combined with or without setting GPIO2 as RFsensor.

What is missing or done wrong by me?

thx in advance!

**Comment by Jason2866** [comment, marked as answer]
Rf received data telegrams are not available. Hardware design of the board.
